src/main/java/org/broadinstitute/hellbender/tools/copynumber/GermlineCNVCaller.java

```diff
@@ -0,0 +1,408 @@
+            specifiedIntervals = new SimpleIntervalCollection(specifiedIntervalsFile);
+        } else {
+            //get sequence dictionary and intervals from the first read-count file to use to validate remaining files
+            //(this first file is read again below, which is slightly inefficient but is probably not worth the extra code)
+            final File firstReadCountFile = inputReadCountFiles.get(0);
+            final SimpleCountCollection firstReadCounts = SimpleCountCollection.read(firstReadCountFile);
+            final SAMSequenceDictionary sequenceDictionary = firstReadCounts.getMetadata().getSequenceDictionary();
+            final LocatableMetadata metadata = new SimpleLocatableMetadata(sequenceDictionary);
+
+            if (intervalArgumentCollection.intervalsSpecified()) {
+                logger.info("Intervals specified...");
+                CopyNumberArgumentValidationUtils.validateIntervalArgumentCollection(intervalArgumentCollection);
+                specifiedIntervals = new SimpleIntervalCollection(metadata,
+                        intervalArgumentCollection.getIntervals(sequenceDictionary));
+            } else {
+                logger.info(String.format("Retrieving intervals from first read-count file (%s)...",
+                        firstReadCountFile));
+                specifiedIntervals = new SimpleIntervalCollection(metadata, firstReadCounts.getIntervals());
+            }
+
+            //in cohort runMode, intervals are specified via -L; we write them to a temporary file
+            specifiedIntervalsFile = IOUtils.createTempFile("intervals", ".tsv");
+            //get GC content (null if not provided)
+            final AnnotatedIntervalCollection subsetAnnotatedIntervals =
+                    CopyNumberArgumentValidationUtils.validateAnnotatedIntervalsSubset(
+                            inputAnnotatedIntervalsFile, specifiedIntervals, logger);
+            if (subsetAnnotatedIntervals != null) {
+                subsetAnnotatedIntervals.write(specifiedIntervalsFile);
+            } else {
+                specifiedIntervals.write(specifiedIntervalsFile);
+            }
+        }
+
+        if (runMode.equals(RunMode.COHORT)) {
+            logger.info("Running the tool in the COHORT mode...");
+            Utils.validateArg(inputReadCountFiles.size() > 1, "At least two samples must be provided in the " +
+                    "COHORT mode");
+            if (inputModelDir != null) {
+                logger.info("(advanced feature) A denoising-model directory is provided in the COHORT run mode; " +
+                        "using the model for initialization and ignoring specified and/or annotated intervals.");
+            }
+        } else { // case run-mode
+            logger.info("Running the tool in the CASE mode...");
+            Utils.validateArg(inputModelDir != null, "An input denoising-model directory must be provided in the " +
+                    "CASE run-mode.");
```

Comment:
`run-mode` -> `mode` here and elsewhere.

Reply:
Done.